internal/config/config.go

```diff
@@ -462,6 +466,14 @@ func ChangeRequiresRestart(from, to Configuration) bool {
 	return false
 }
 
+func convertV10V11(cfg *Configuration) {
+	// Set minimum disk space on existing folders to 500MB
+	for i := range cfg.Folders {
+		cfg.Folders[i].MinimumDiskSpaceMB = 500
```

Comment:
Not sure I like going by MB and defaulting to this. What about when if we sync 10 MB of files into an empty 256 MB flash card on an embedded system - shouldn't that work out of the box?